scripts/lunch.coffee

```diff
@@ -93,12 +94,30 @@ module.exports = (robot) ->
 
   robot.respond /lunch locations$/i, (msg) ->
 
+    # sort by least latest visit then by least count
+    sort_lunch = (lunches) ->
+
+      least_latest_visit = (a, b) ->
+        # reset time to only compare dates
+        date_a = new Date(a.at).setHours(0, 0, 0, 0).valueOf()
+        date_b = new Date(b.at).setHours(0, 0, 0, 0).valueOf()
+        if date_a.valueOf() > date_b.valueOf() then return 1
```

Comment:
You can probably just return `date_a.valueOf() - date_b.valueOf()` right? Usually it doesn't need to be *exactly* +1 or -1; +3, -5, those work fine (I think?)